test/core/event_engine/test_suite/BUILD

```diff
@@ -82,6 +83,8 @@ grpc_cc_test(
     uses_polling = False,
```

Comment:
This fails with `GRPC_POLL_STRATEGY=none`

https://source.cloud.google.com/results/invocations/4cead0a6-9484-4fd7-80ec-2fac56cf8746/targets/github%2Fgrpc%2Frun_tests%2Fcpp_macos_dbg_native/tests

CC: @drfloob

Reply:
I see, thanks.